man/os-release.xml

```diff
@@ -350,6 +350,16 @@
           <literal>BUG_REPORT_URL="https://bugzilla.redhat.com/"</literal>.</para></listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>SUPPORT_END=</varname></term>
+
+          <listitem><para>The date at which support for this version of the OS ends. (What exactly "lack of
+          support" means varies between vendors, but generally users should assume that updates, including
+          security fixes, will not be provided.) The value is a date in the format
+          <literal>YYYY-MM-DD</literal>, and specifies the last day on which support <emphasis>is</emphasis>
```

Comment:
hmm, i guess there's a bikeshedding discussion to be had about the picked date format.

Could also be unix seconds? Or ISO?

I guess the format you picked is a subset of what systemd.time(5) describes, which I guess is a nice property. So I guess this is fine to pick this one.

Reply:
This is ISO 8601 date format and imho the only sane way to define this.